Framework/DataHandling/src/MeshFileIO.cpp

```diff
@@ -5,26 +5,11 @@
 //   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 // SPDX - License - Identifier: GPL - 3.0 +
 #include "MantidDataHandling/MeshFileIO.h"
+#include "MantidGeometry/Objects/ShapeFactory.h"
```

Comment:
I think the cppcheck failure is due to this although this code does work

Reply:
https://builds.mantidproject.org/job/pull_requests-cppcheck/47570/cppcheckResult/source.52/